guide/contribute/contribute.md

```diff
@@ -61,14 +60,14 @@ Tips on how to write for the guide. The goal is to achieve a consistent tone acr
 
 **[Illustration guidelines]({{ '/guide/contribute/illustration-guidelines/' | relative_url }})**
 
-Notes on how to prepare visuals for use in the guide. We encourage diverse creative expressions, so this page focuses mostly on basics like file names and image sizes.
+Creating visuals for the Bitcoin Design Guide? We recommend following these best practices for file naming, image sizing, and on.
 
 ---
 
 **[Formatting]({{ '/guide/contribute/formatting/' | relative_url }})**
 
-Layout and formatting options available for content pages, from headers and blockquotes to images and embeds.
+Layout and formatting recommendations from headers and blockquotes to images and embeds.
```

Comment:
`recommendations` isn't quite right, as the page lists what's technically possible. It's more of a technical documentation page.

Reply:
Nice catch, will change it to `options`.

Reply:
https://github.com/BitcoinDesign/Guide/pull/224/commits/f6e588140138147c26ecf4f38994d4b1eab80dc5